tests/test_public_interface.py

```diff
@@ -388,3 +388,39 @@ def test_copy_delta_table(tmp_path):
 
     chispa.assert_df_equality(origin_details, copied_details)
     chispa.assert_df_equality(origin_table.toDF(), copied_table.toDF(), ignore_row_order=True)
+
+
+
+def test_append_without_duplicates(tmp_path):
+    path = f"{tmp_path}/append_without_duplicates"
+    data = [
+        (1,"A","B"),
+        (2,"R","T"),
+        (3,"X","Y")
+    ]
+    df = spark.createDataFrame(data, ["col1","col2","col3"])
+    df.write.format("delta").save(path)
+
+    deltaTable = DeltaTable.forPath(spark, path)
+
+    append_data = spark.createDataFrame(
```

Comment:
This test data doesn't have any data that's already in the Delta table.  We need some duplicate data here to demonstrate that the duplicate data isn't appended, only the new data.

Reply:
@MrPowers Added the multiple column test also. Results below: 
<img width="1620" alt="image" src="https://user-images.githubusercontent.com/98817232/208475642-ae6b33ff-af68-4900-b6ef-a1129485167f.png">